Snippets/compact_gpos.py

```diff
@@ -0,0 +1,124 @@
+import argparse
```

Comment:
who's the potential user for this script? maybe add a docstring summarizing what this script does, what's it for -- for our future selves.

Reply:
Done: https://github.com/fonttools/fonttools/pull/2326/files#diff-931862a8bb3134a124ab6c069eeee5a0f14905f5e91a0d3c10b3f622a6638d97R3-R19